.vsts-ci.yml

```diff
@@ -56,6 +56,30 @@ jobs:
        CMAKE_OPTIONS=-DUSE_HTTPS=mbedTLS -DSHA1_BACKEND=mbedTLS
        LEAK_CHECK=valgrind
 
+- job: linux_xenial_arm32
+  displayName: 'Linux (Xenial; arm32)'
+  pool:
+    vmImage: 'Ubuntu 16.04'
+  steps:
+  - template: ci/vsts-docker.yml
+    parameters:
+      qemu: 'true'
+      imageName: 'libgit2/xenial-arm32:test'
+      environmentVariables: |
+       SKIP_PROXY_TESTS=true
+
+- job: linux_xenial_arm64
+  displayName: 'Linux (Xenial; arm64)'
+  pool:
+    vmImage: 'Ubuntu 16.04'
+  steps:
+  - template: ci/vsts-docker.yml
+    parameters:
+      qemu: 'true'
+      imageName: 'libgit2/xenial-arm64:test'
+      environmentVariables: |
+       SKIP_PROXY_TESTS=true
```

Comment:
Is this all supposed to be part of our "main" CI infrastructure? If so, by how much does it increase our CI build times? If the delay is quite noticable, I'd vote to have this as a nightly job instead, as we have said in past discussions.

Reply:
I 100% agree.  I was just hoping to iterate on these by putting them in the PR validation configuration so that they would get built.  (There are a handful of other ways to do this, I could have set up a new pipelines configuration, but I'm lazy.)